addons/web_editor/static/src/js/editor/odoo-editor/src/commands/commands.js

```diff
@@ -987,10 +988,7 @@ export const editorCommands = {
     insertHorizontalRule(editor) {
         const selection = editor.document.getSelection();
         const range = selection.getRangeAt(0);
-        const element = closestElement(
-            range.startContainer,
-            'P, PRE, H1, H2, H3, H4, H5, H6, BLOCKQUOTE',
-        );
+        const element = closestElement(range.startContainer, paragraphRelatedElements) || closestBlock(range.startContainer);
```

Comment:
No test?

Reply:
Added the testcases.

You mentioned in the task pad to change it to `const element = closestElement(range.startContainer, paragraphRelatedElements) || closestBlock(range.startContainer);` but isn't it same as `const element = closestBlock(range.startContainer);` or am I not seeing something that you are.

Reply:
It's not the same thing because you can be a block without being a `paragrahRelatedElement`.